test/tstQueryTreeManufacturedSolution.cpp

```diff
@@ -88,11 +92,13 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(structured_grid, TreeTypeTraits,
   Kokkos::deep_copy(offset_host, offset);
 
   // we expect the collision list to be diag(0, 1, ..., nx*ny*nz-1)
-  for (int i = 0; i < n; ++i)
-  {
-    BOOST_TEST(indices_host(i) == i);
-    BOOST_TEST(offset_host(i) == i);
-  }
+  std::vector<int> indices_ref(n);
+  std::vector<int> offset_ref(n + 1);
+  std::iota(indices_ref.begin(), indices_ref.end(), 0);
+  std::iota(offset_ref.begin(), offset_ref.end(), 0);
+  BOOST_TEST(make_compressed_storage(offset_host, indices_host) ==
+                 make_compressed_storage(offset_ref, indices_ref),
+             tt::per_element());
```

Comment:
IMHO, the original version is easier to read.

Reply:
IMNSHO, the old version clutters the output log

Reply:
I actually wish Andrey went all the way and actually cleaned up the entire unit test so we only do
```C++
BOOST_TEST_QUERY_TREE(ExecutionSpace{}, tree, predicates, make_reference_solution(indices_ref, offset_ref));

Reply:
Look closer, it was a poor man's compare results but they will be unsorted

Reply:
Addressed.